dashboard/app/models/programming_expression.rb

```diff
@@ -28,7 +28,7 @@ class ProgrammingExpression < ApplicationRecord
 
   serialized_attrs %w(
     color
-    parameters
+    syntax
```

Comment:
the name `syntax` seems a bit general. can you describe how you envision this being used? For example, will this control how the block appears in the toolbox, in addition to lesson plans and rollup pages?

Reply:
Syntax is what we call the field on CB right now so I was matching that. https://curriculum.code.org/docs/applab/onEvent/ . It is currently used to display what the general case for the command is. Nothing in our programming_expressions impacts the toolbox at all right now. I think until we better understand how to show droplet and blocky blocks this will be used as a substitute for showing the blocks as real blocks. It also will probably continue to be used like in the example above